docs/manuals/source/IntroductionAndTutorial/InstallingBareos.rst

```diff
@@ -59,6 +59,26 @@ The following code snippets are shell scripts that can be used as orientation ho
 **See** https://download.bareos.org/bareos/release/ **for applicable releases and distributions.**
 See :ref:`section-UniversalLinuxClient` if your distribution is not present.
 
+To simplify the, all Linux distribution and FreeBSD repositories on https://download.bareos.org/ and https://download.bareos.com/ contain a script named :file:`add_bareos_repositories.sh`.
+
+Download the :file:`add_bareos_repositories.sh` script
+matching the requested Bareos release
+and the distribution of the target system.
```

Comment:
```suggestion
Download the :file:`add_bareos_repositories.sh` script matching the requested Bareos release and the distribution of the target system.
```

Reply:
You have not changed the text, havn't you?